<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="play_Btn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>510</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="last_Btn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>520</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>上一曲</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="next_Btn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>520</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>下一曲</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="volumeControl">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>500</y>
     <width>22</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="list">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>321</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="muteButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>590</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="add_Btn">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="music_name_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>570</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playPattern_Btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="playList_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>590</y>
     <width>331</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="nowplay_label">
      <property name="text">
       <string>正在播放：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="now_time_label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="seekSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="time_label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="search_LineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="search_Btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_Btn">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>0</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="min_Btn">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>0</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="logo_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="Lrc_list">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>561</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tital_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>好音乐 感谢聆听...  </string>
   </property>
  </widget>
  <widget class="QLabel" name="lrcList_label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>歌词：</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>search_LineEdit</tabstop>
  <tabstop>search_Btn</tabstop>
  <tabstop>list</tabstop>
  <tabstop>play_Btn</tabstop>
  <tabstop>last_Btn</tabstop>
  <tabstop>next_Btn</tabstop>
  <tabstop>seekSlider</tabstop>
  <tabstop>volumeControl</tabstop>
  <tabstop>muteButton</tabstop>
  <tabstop>playPattern_Btn</tabstop>
  <tabstop>add_Btn</tabstop>
  <tabstop>min_Btn</tabstop>
  <tabstop>close_Btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
